week1/homework/src/server.js

```diff
@@ -10,11 +10,46 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    if (request.url === '/state') {
+      response.writeHead(200, {
+        'Content-Type': 'application/json'
+      });
+      response.end(JSON.stringify({
+        state: state
+      }));
+    } else if (request.url === '/add') {
+      response.writeHead(200, {
+        'Content-Type': 'application/json'
+      });
+      response.end(JSON.stringify({
+        state: state++
```

Comment:
You are returning the `state` variable before you increment it. That's why the corresponding test doesn't pass.